onnxconverter_common/optimizer.py

```diff
@@ -1402,11 +1304,105 @@ def apply(self, node_list):
         return node_list, True
 
 
-class PushTransposeOptimizer(object):
-    opt_number = 0
+class TransposeOptimizer(object):
+    @staticmethod
+    def find_1(node):
```

Comment:
can you find a better name?

Reply:
Use names: `find_local` and `find_push_down`.